src/lightly_train/_transforms/task_transform.py

```diff
@@ -35,5 +48,5 @@ def __init__(self, transform_args: TaskTransformArgs):
             )
         self.transform_args = transform_args
 
-    def __call__(self, input: TaskTransformInput) -> TaskTransformOutput:
+    def __call__(self, input: Any) -> Any:
```

Comment:
Does mypy complain about the following? Functions are usually covariant in the output type so this shouldn't be a problem.

```suggestion
    def __call__(self, input: Any) -> TaskTransformOutput:
```

Reply:
Fully correct. Changed it accordingly.